contrib/init/upstart/docker.conf

```diff
@@ -34,12 +34,12 @@ end script
 
 script
 	# modify these in /etc/default/$UPSTART_JOB (/etc/default/docker)
-	DOCKER=/usr/bin/$UPSTART_JOB
+	DOCKERD=/usr/bin/$UPSTART_JOB
```

Comment:
Oh, and `$UPSTART_JOB` here should be `dockerd` instead (like the `$BASE` change above for sysvinit).